README.adoc

```diff
@@ -190,8 +235,12 @@ include::finish/resources/CustomConfigSource.json[tags=**]
 [role='command']
 include::{common-includes}/mvnbuild.adoc[]
 
-While the server is running, navigate to the http://localhost:9080/health[http://localhost:9080/health^] URL to find the the health
-report on the two services.
+While the server is running, navigate to the http://localhost:9080/health[http://localhost:9080/health^] URL to find
+the aggregated liveness and readiness health reports on the two services.
+
+You can also navigate to http://localhost:9080/health/ready[http://localhost:9080/health/ready^]
+URL to view the readiness health report or the http://localhost:9080/health/live[http://localhost:9080/health/live^] 
+URL to view the liveness health report.
```

Comment:
Change
"You can also navigate to http://localhost:9080/health/ready[http://localhost:9080/health/ready^] URL to view the readiness health report or the http://localhost:9080/health/live[http://localhost:9080/health/live^] URL to view the liveness health report."
To
"You can also navigate to **the** http://localhost:9080/health/ready[http://localhost:9080/health/ready^]
URL to view the readiness health report, or the http://localhost:9080/health/live[http://localhost:9080/health/live^] URL to view the liveness health report."